packages/bot/src/components/run-panel.jsx

```diff
@@ -76,20 +79,20 @@ const drawerFooter = ({
                 is_open={is_dialog_visible}
                 closeModal={closeModal}
             >
-                {translate('Please log in.')}
+                {dialog_content}
             </Dialog>
             <InfoOutlineIcon className='run-panel__icon-info' />
         </div>
     );
 };
 
 class RunPanel extends React.PureComponent {
-    componentWillUnmount() {
-        this.props.onUnmount();
-    }
+    // componentWillUnmount() {
+    //     this.props.onUnmount(); TODO: Dispose of listeners.
+    // }
 
     render() {
-        const content = drawerContent();
+        const content = drawerContent(this.props);
```

Comment:
@aaron-binary you just need to pass two props to drawerContent, 
```suggestion
        { active_index, setActiveTabIndex } = this.props;
        const content = drawerContent({ active_index, setActiveTabIndex });
```